v19.2/start-a-local-cluster-in-docker-windows.md

```diff
@@ -26,171 +26,179 @@ Running multiple nodes on a single host is useful for testing CockroachDB, but i
 
 Since you'll be running multiple Docker containers on a single host, with one CockroachDB node per container, you need to create what Docker refers to as a [bridge network](https://docs.docker.com/engine/userguide/networking/#/a-bridge-network). The bridge network will enable the containers to communicate as a single cluster while keeping them isolated from external networks.
 
-~~~ powershell
+```powershell
 PS C:\Users\username> docker network create -d bridge roachnet
-~~~
+```
 
 We've used `roachnet` as the network name here and in subsequent steps, but feel free to give your network any name you like.
 
 ## Step 2. Start the cluster
 
 1. Start the first node:
 
-    {{site.data.alerts.callout_info}}Be sure to replace <code>&#60;username&#62;</code> in the <code>-v</code> flag with your actual username.{{site.data.alerts.end}}
+   {{site.data.alerts.callout_info}}Be sure to replace <code>&#60;username&#62;</code> in the <code>-v</code> flag with your actual username.{{site.data.alerts.end}}
 
-    ~~~ powershell
-    PS C:\Users\username> docker run -d `
-    --name=roach1 `
-    --hostname=roach1 `
-    --net=roachnet `
-    -p 26257:26257 -p 8080:8080  `
-    -v "//c/Users/<username>/cockroach-data/roach1:/cockroach/cockroach-data"  `
-    {{page.release_info.docker_image}}:{{page.release_info.version}} start `
-    --insecure `
-    --join=roach1,roach2,roach3
-    ~~~
+   ```powershell
```

Comment:
I'm not sure why we need to change ~~~ to ``` in the doc. Mind sharing your reasoning?

Reply:
The only changes I deliberately made were lines 233-241 (add/remove). 🙂 If other lines changed, it was not intentional and could be my lack of experience here. I am more than happy to fix/revert as needed. 👍

Reply:
Oh..no problem! Yes, please revert the changes except the one with the wrong command. Thanks!

Reply:
Will do. These changes are needed with 20.1 and 20.2 as well, yes?

Reply:
🤦🏻‍♂️ It was some settings on my code editor...

Reply:
ahh..I see. Yes, the 20.1 and 20.2 docs need to be updated as well.

Reply:
👍 Should be fixed here in a few.